content/blog/2018-06-26-roomba.Rmd

```diff
@@ -0,0 +1,107 @@
+---
+slug: "roomba"
+title: A package for tidying nested lists
+package_version: 0.0.1
+authors:
+  - name: Amanda Dobbyn
+    url: https://github.com/aedobbyn
+  - name: Jim Hester
+    url: https://github.com/jimhester
+  - name: Laura DeCicco
+    url: https://github.com/ldecicco-USGS
+  - name: Christine Stawitz
+    url: https://github.com/cstawitz
+  - name: Isabella Velasquez 
+    url: https://github.com/ivelasq
+date: 2018-06-26
+categories: blog
+topicid: 925

```

Comment:
s/topicid: 925/topicid/

Reply:
I think this means leave the topicid blank? I removed 925 :)